btree/btree.c

```diff
@@ -621,6 +624,65 @@ enum {
 	INTERNAL_NODE_VALUE_SIZE = sizeof(void *),
 };
 
+#define M0_BTREE_TX_CAPTURE(tx, seg, ptr, size)                         \
+			   m0_be_tx_capture(tx, &M0_BE_REG(seg, size, ptr))
+
+#if AVOID_BE_SEGMENT
+
+#undef M0_BTREE_TX_CAPTURE
+#define M0_BTREE_TX_CAPTURE(tx, seg, ptr, size)                              \
+	do { (tx) = (tx); (seg) = (seg); (ptr) = (ptr); } while (0)
+
+#undef M0_BE_ALLOC_ALIGN_BUF_SYNC
+#define M0_BE_ALLOC_ALIGN_BUF_SYNC(buf, shift, seg, tx)                      \
```

Comment:
`seg and tx` are not used in the macro expansion.

Reply:
Yes they are not used in this dummy macro but the macro in alloc.h will use the seg and tx hence we have those parameters in the dummy macro to match the signatures and avoid changing the code later.